notebooks/community/model_garden/model_garden_mediapipe_gesture_recognition.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    layer_widths: str = ""  # @param {type:"string"}
</pre><p>Should this be a list?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>